src/celeritas/grid/ValueGridBuilder.hh

```diff
@@ -66,7 +66,7 @@ class ValueGridXsBuilder final : public ValueGridBuilder
   public:
     //!@{
     //! \name Type aliases
-    using SpanConstDbl = Span<double const>;
+    using SpanConstDbl = LdgSpan<double const>;
```

Comment:
These classes are all host-only (they're used in construction) so I don't think they should be modified.